### PR TITLE
Clarify docs that bss is zero-initialized

### PR DESCRIPTION
**Pull request by jmr**
Addresses comment raised in #1202.

### CODE REVIEW

**Review by oliverschmidt — CHANGES_REQUESTED** [review]

**Comment by oliverschmidt** [comment]
I personally would suggest to leave ld65.sgml alone.

File: doc/ld65.sgml [hunk 1]
```diff
@@ -533,23 +533,23 @@ specify a segment attribute. There are five possible segment attributes:
 <tscreen><verb>
         ro          means readonly
         rw          means read/write
-        bss         means that this is an uninitialized segment
+        bss         means that this is a zero-initialized segment
```
[the remark this answers is not in the document]
Comment (jmr) [reply doc/ld65.sgml]: reverted

File: doc/ld65.sgml [hunk 2]
```diff
@@ -533,23 +533,23 @@ specify a segment attribute. There are five possible segment attributes:
...
         zp          a zeropage segment
         overwrite   a segment that overwrites (parts of) another one
 
 </verb></tscreen>
 
 So, because we specified that the segment with the name BSS is of type bss,
-the linker knows that this is uninitialized data, and will not write it to an
+the linker knows that this is zero-initialized data, and will not write it to an
```
[the remark this answers is not in the document]
Comment (jmr) [reply doc/ld65.sgml]: reverted

File: doc/ld65.sgml [hunk 3]
```diff
@@ -533,23 +533,23 @@ specify a segment attribute. There are five possible segment attributes:
...
 output file. This is an important point: For the assembler, the <tt/BSS/
 segment has no special meaning. You specify, which segments have the bss
 attribute when linking. This approach is much more flexible than having one
 fixed bss segment, and is a result of the design decision to supporting an
 arbitrary segment count.
 
 If you specify "<tt/type = bss/" for a segment, the linker will make sure that
-this segment does only contain uninitialized data (that is, zeroes), and issue
-a warning if this is not the case.
+this segment only contains zero-initialized data, and issue a warning if this
```
[the remark this answers is not in the document]
Comment (jmr) [reply doc/ld65.sgml]: reverted

**Review by oliverschmidt — APPROVED** [review]

**Pull request merged**